tests/drivers/counter/counter_basic_api/boards/nrf9160_pca10090.conf

```diff
@@ -0,0 +1,4 @@
+CONFIG_COUNTER_TIMER0=y
```

Comment:
You need to add license and copyright here

Reply:
I usually don't see licenses in `.conf` files.